pygsti/protocols/rpe.py

```diff
@@ -323,3 +323,9 @@ def raw_angles(self):
 RPEDesign = RobustPhaseEstimationDesign
 RPE = RobustPhaseEstimation
 RPEResults = RobustPhaseEstimationResults
+
+__all__ = [
```

Comment:
Why is this needed?

Reply:
It's included in this patch just so that autodoc doesn't try to document external names imported in `protocols` from `protocols.rpe`, namely `OrderedDict`, `namedtuple`, `Namespace` and `numpy`.

There are less-invasive ways of doing this but I figure we don't want to expose those anyway so we might as well.

We could also import those names as protected in `protocols.rpe` to match the pattern used elsewhere but I felt that was more invasive.

Reply:
Gotcha - If you don't mind, let's switch over to using protected names like we do elsewhere and drop the explicit __all__ assignment.  It's a good kind of invasive :)

Reply:
Sounds good to me. Resolved with 1c078e8